targets/unity-v2/source/Entity/ScreenTimeTracker.cs

```diff
@@ -151,7 +163,12 @@ public void Send()
                     eventsInTheBatch++;
```

Comment:
I think eventsInTheBatch is redundant.
It's just a mirror of:
request.Events.Count()

Reply:
fixed